src/registrar/admin.py

```diff
@@ -50,13 +50,40 @@ class MyHostAdmin(AuditedAdmin):
     inlines = [HostIPInline]
 
 
+class DomainAdmin(AuditedAdmin):
+
+    """Custom domain admin class to add extra buttons."""
+
+    change_form_template = "django/admin/domain_change_form.html"
+    readonly_fields = ["state"]
+
+    def response_change(self, request, obj):
+        if "_place_client_hold" in request.POST:
```

Comment:
**comment (non-blocking)**: Because this is a magic string that has to match with one in a "distant" template file, maybe a comment on where this value comes from?

Reply:
+1 to that.

Reply:
Fixed.